### PR TITLE
Explanation of suggested add-ons

### PR DESCRIPTION
**Pull request by andrewfg**
Added information about the new add-on suggestion finder feature.

Note: I also changed one of the screenshot images; but am not sure if that is also automatically included in the merge.

Signed-off-by: Andrew Fiddian-Green <software@whitebear.ch>

### CODE REVIEW

**Comment by andrewfg** [comment]
@lolodomo you want to merge this in v4.1

File: tutorials/getting_started/first_steps.md [hunk 1]
```diff
@@ -31,13 +31,18 @@ Start by setting your language, region, and time zone. You can also set your loc
 ### Install add-ons
 
 You now have the option to install openHAB add-ons.
-Do this if you already know that you'll need specific add-ons (e.g. if you're upgrading from an older openHAB system), or skip this step if you prefer to install add-ons individually.
+The openHAB system does an automatic scan to discover home automation equipment and software on your PC and LAN, and based the result it recommends suggested add-ons to install.
```
Comment (stefan-hoehn) [inline tutorials/getting_started/first_steps.md:34]: "and based the result it recommends suggested add-ons to install." - I think there is something wrong with that sentence, IMHO

Either 
"and based >on< the result it recommends suggested add-ons to install."

or
"and suggests installing add-ons based on the result."

File: tutorials/getting_started/first_steps.md [hunk 2]
```diff
@@ -31,13 +31,18 @@ Start by setting your language, region, and time zone. You can also set your loc
 ### Install add-ons
 
 You now have the option to install openHAB add-ons.
-Do this if you already know that you'll need specific add-ons (e.g. if you're upgrading from an older openHAB system), or skip this step if you prefer to install add-ons individually.
+The openHAB system does an automatic scan to discover home automation equipment and software on your PC and LAN, and based the result it recommends suggested add-ons to install. 
+You can modify the list of add-ons.
+However, do this only if you already know that you'll need specific add-ons (e.g. if you're upgrading from an older openHAB system).
```
Comment (stefan-hoehn) [inline tutorials/getting_started/first_steps.md:36]: I would prefer: you'll -> you will

File: tutorials/getting_started/first_steps.md [hunk 3]
```diff
@@ -31,13 +31,18 @@ Start by setting your language, region, and time zone. You can also set your loc
...
+Or skip this step if you prefer to install add-ons individually later.
 You can always install/remove add-ons in the future, enabling you to modify and improve your system over time.
 
-![addons wizard](images/wizard_addons.png)
+![addons wizard](https://github.com/openhab/openhab-docs/assets/893994/ecbd893b-3b95-4fd6-a89c-6f9a87313229)
```
Comment (stefan-hoehn) [inline tutorials/getting_started/first_steps.md:40]: Is that a stable link? We are we using this particular link?

**Review by stefan-hoehn — CHANGES_REQUESTED** [review]

File: tutorials/getting_started/first_steps.md (hunk 3 above)
Comment (andrewfg) [reply tutorials/getting_started/first_steps.md:40]: Same question as for my other PR => @Confectrician

Comment (andrewfg) [reply tutorials/getting_started/first_steps.md:41]: Here is the image

image

Comment (andrewfg) [reply tutorials/getting_started/first_steps.md]: I fixed the image link

**Review by stefan-hoehn — APPROVED** [review]
I am feeling much better with that, Andrew, thanks a lot.

**Pull request merged**